scipy/stats/tests/test_stats.py

```diff
@@ -6778,7 +6778,7 @@ def check_equal_pmean(array_like, exp, desired, axis=None, dtype=None,
     assert_equal(x.dtype, dtype)
 
 
-class TestHarMean:
+class TestHMean:
```

Comment:
While we're here, let's make these easier to find by following a more predictable naming convention.